packages/react/src/components/header/Header.stories.tsx

```diff
@@ -154,6 +154,128 @@ export const WithFullFeatures = (args) => (
   </>
 );
 
+export const WithFullFeaturesDarkTheme = (args) => (
+  <>
+    <Header {...args} onDidChangeLanguage={languageChangedAction}>
+      <Header.UniversalBar primaryLinkText="Helsingin kaupunki" primaryLinkHref="#">
```

Comment:
Maybe add the SkipLink in here so we and users can see how it changes? It doesn't change much, but the blue focus colour is removed in dark theme.

Reply:
Now there's only one story which covers all features for header dark theme.